test/cutting/cut_finding/test_cut_finder_results.py

```diff
@@ -189,9 +188,7 @@ def test_four_qubit_circuit_two_qubit_qpu(
         interface.export_subcircuits_as_string(name_mapping="default") == "AABB"
     )  # circuit separated into 2 subcircuits.
 
-    assert (
-        optimization_pass.get_stats()["CutOptimization"] == array([15, 46, 15, 6])
-    ).all()  # matches known stats.
+    assert optimization_pass.get_stats()["CutOptimization"][3] <= settings.max_backjumps
```

Comment:
Why only consider the element at index `[3]`?

Reply:
It's really the only number (the number of backjumps) that is possible to (read easiest to) predict and constrain.

Reply:
I see now: `get_stats` returns an array but every element of it means something specific.  This would have been better as a data structure or `namedtuple` rather than a numpy array.

Reply:
'Tis done ([837875c](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/585/commits/837875c30bd056c202470426924b2dedb166d307)). Have also added a release note.

Reply:
> 'Tis done ([837875c](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/585/commits/837875c30bd056c202470426924b2dedb166d307)). Have also added a release note.

Thank you.

We should probably actually revert this on this branch and then do this as a separate PR.  That way, we can backport this current PR since the interface does not change.  The improved interface can be done in 0.8.0 with a release note.

Reply:
Hmm this function isn't actually exposed in the API. Does that matter?

Reply:
> Hmm this function isn't actually exposed through the API. Does that matter?

OK, in that case back-porting should be fine as is.

Sorry, I thought I saw something previously in the notebook where these same four numbers turned up, but that is either gone now or a phantom memory.